src/radical/entk/appman/appmanager.py

```diff
@@ -75,7 +75,8 @@ def __init__(self,
                  rts=None,
                  rmq_cleanup=None,
                  rts_config=None,
-                 name=None):
+                 name=None,
+                 config={}):
```

Comment:
@lee212: I'm afraid you can't use complex types as default values in Python.  Consider the following script:
```py
#!/usr/bin/env python3

# ------------------------------------------------------------------------------
#
def test(cfg={}):
    if not cfg:
        print('no  cfg')
        cfg['foo'] = 'bar'
    else:
        print('has cfg: %s' % cfg)


# ------------------------------------------------------------------------------
#
if __name__ == '__main__':

    test()
    test()


# ------------------------------------------------------------------------------

```
which has the unexpected result:
```sh
$ ./t.py
no  cfg
has cfg: {'foo': 'bar'}
```

You will have to change that to:
```py
def test(cfg=None):
    if cfg is None:
        cfg = dict()
```

Reply:
Thanks for the comment which makes sense. I applied this.